web/themes/custom/hoeringsportal/assets/js/image-fade.js

```diff
@@ -0,0 +1,26 @@
+require("../css/module/_image-fade.scss");
+
+let imgFadeSelector = document.querySelector(".image-fade");
+let firstImage = document.getElementById("img-0");
+
+if (imgFadeSelector && firstImage) {
+  setInterval(nextImage, imgFadeSelector.dataset.imageDuration ?? 15000);
+  firstImage.classList.add("show");
+}
+
+let curImage = 0;
+let numImages = document.querySelectorAll(".image-fade img").length;
+
+function nextImage() {
+  // Hide old image
+  document.getElementById("img-" + curImage).classList.toggle("show");
+
+  // Compute next image
+  curImage++;
+  if (curImage > numImages - 1) {
+    curImage = 0;
+  }
+
+  // Show next image
+  document.getElementById("img-" + curImage).classList.toggle("show");
+}
```

Comment:
This works, but a more clean and easy to understand solution may be something like https://codepen.io/rimi-itk/pen/LEGvPjQ.